azure-pipelines.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       matrix:
         # We're only running one platform per Python version to speed up builds
         Python36Linux:
-          imageName: "ubuntu-latest"
+          imageName: "ubuntu-20.04"
```

Comment:
I just merged the updates from `master` to `v4` which contain this commit: https://github.com/explosion/spaCy/commit/8267aa1b65783ea29e42f470eaf7553ff7508206
So it's probably best if you merge in the latest `v4` again so we don't end up with merge conflicts :-)